src/EnergyPlus/InputProcessing/CsvParser.cc

```diff
@@ -233,53 +201,49 @@ void CsvParser::parse_header(std::string_view csv, size_t &index, bool &success,
             next_token(csv, index);
         } else {
             header.push_back(parse_value(csv, index));
-            if (!success) return;
         }
     }
 }
 
 void CsvParser::parse_line(std::string_view csv, size_t &index, json &columns)
 {
     Token token;
-    int column_num = 0;
+    size_t column_num = 0;
+    size_t parsed_values = 0;
+    const size_t num_columns = columns.size(); // Csv isn't empty, so we know it's at least 1
+
+    size_t this_cur_line_num = cur_line_num;
+    size_t this_beginning_of_line_index = beginning_of_line_index;
```

Comment:
Here I get the indices I'll need to format the error, and I calculate the expected number of columns